src/AuthorizedSocketConnection.js

```diff
@@ -156,14 +110,14 @@ export default class TenantSocketConnection {
       document: parse(query),
       variableValues: variables,
       contextValue: {
-        httpApi: this.config.context.httpApi,
+        ...this.config.context,
```

Comment:
trying get this to work with `node-resource` so this is more opaque

Reply:
yeah, looks good